cockroachdb/datadog_checks/cockroachdb/metrics.py

```diff
@@ -264,6 +264,79 @@
+    'changefeed_running': 'changefeed.running',
+    'jobs_changefeed_resume_retry_error': 'jobs.changefeed.resume.retry.error',
+    'ranges_overreplicated': 'ranges.overreplicated',
+    'rocksdb.compactions.total': 'rocksdb.compactions',
+    'sql_conn_latency': 'sql.conn.latency',
+    'sql_distsql_contended_queries_count': 'sql.distsql.contended.queries',
+    'sql_failure_count': 'sql.failure',
+    'sql_full_scan_count': 'sql.full.scan',
+    'sql_statements_active': 'sql.statements.active',
+    'sql_txn_latency': 'sql.txn.latency',
+    'sql_txns_open': 'sql.txns.open',
+    'sys_cpu_combined_percent_normalized': 'sys.cpu.combined.percent.normalized',
+    'sys_host_net_recv_bytes': 'sys.host.net.recv.bytes',
+    'sys_host_net_send_bytes': 'sys.host.net.send.bytes',
+    'sys_host_disk_read_bytes': 'sys.host.disk.read.bytes',
+    'sys_host_disk_write_bytes': 'sys.host.disk.write.bytes',
+    'admission_requested_kv': 'admission.requested.kv',
+    'admission_requested_kv_stores': 'admission.requested.kv.stores',
+    'admission_admitted_sql_sql_response': 'admission.admitted.sql.sql.response',
+    'admission_admitted_sql_leaf_start': 'admission.admitted.sql.leaf.start',
+    'admission_granter_total_slots_kv': 'admission.granter.total.slots.kv',
+    'admission_wait_queue_length_sql_sql_response': 'admission.wait.queue.length.sql.sql.response',
+    'admission_granter_used_slots_kv': 'admission.granter.used.slots.kv',
+    'admission_errored_sql_root_start': 'admission.errored.sql.root.start',
+    'admission_wait_queue_length_kv': 'admission.wait.queue.length.kv',
+    'admission_errored_kv': 'admission.errored.kv',
+    'admission_requested_sql_leaf_start': 'admission.requested.sql.leaf.start',
+    'admission_errored_sql_leaf_start': 'admission.errored.sql.leaf.start',
+    'admission_wait_sum_sql_root_start': 'admission.wait.sum.sql.root.start',
+    'admission_admitted_kv': 'admission.admitted.kv',
+    'admission_admitted_sql_kv_response': 'admission.admitted.sql.kv.response',
+    'admission_granter_used_slots_sql_root_start': 'admission.granter.used.slots.sql.root.start',
+    'admission_wait_durations_kv': 'admission.wait.durations.kv',
+    'admission_wait_durations_kv_stores': 'admission.wait.durations.kv.stores',
+    'admission_wait_queue_length_sql_root_start': 'admission.wait.queue.lengths.sql.root.start',
+    'admission_wait_queue_length_sql_kv_response': 'admission.wait.queue.length.sql.kv.response',
+    'admission_wait_durations_sql_leaf_start': 'admission.wait.durations.sql.leaf.start',
+    'admission_admitted_sql_root_start': 'admission.admitted.sql.root.start',
+    'admission_granter_io_tokens_exhausted_duration_kv': 'admission.granter.io.tokens.exhausted.duration.kv',
+    'admission_requested_sql_sql_response': 'admission.requested.sql.sql.response',
+    'admission_wait_durations_sql_sql_response': 'admission.wait.durations.sql.sql.response',
```

Comment:
having `.sql.sql.` looks weird on our side, do you think we can rename it?

Reply:
Yeah good point. I'm not sure what to rename it to because the prometheus description doesn't provide any more specifics for those metrics. I'll check the documentation to see what exactly that component is and rename the metric. Thanks for raising!

Reply:
Seems that the [documentation](https://github.com/cockroachdb/cockroach/blob/master/docs/tech-notes/admission_control.md#kinds-of-work-and-queues) does refer to those as `sql-sql` so I'll rename them to `sql_sql`.